examples/compute/render.wgsl

```diff
@@ -0,0 +1,31 @@
+
+
+struct Renderer {
+  proj_view: mat4x4<f32>;
+};
+
+struct Particle {
+  position : vec4<f32>;
+  velocity : vec4<f32>;
+};
+
+struct Particles {
+  particles : [[stride(32)]] array<Particle, 2000>;
+};
+
+[[group(0), binding(0)]] var<uniform> uRenderer: Renderer;
+[[group(0), binding(1)]] var<uniform> uParticles : Particles;
```

Comment:
Why make this a uniform rather then a storage buffer? With a storage buffer we could keep the array size more flexible

```
particles : [[stride(32)]] array<Particle>;
```

Reply:
I think I found out why you did this. I think perhaps instead of doing it this way we should enable vertex storage buffers with this line:

```rust
features: wgpu::Features::VERTEX_WRITABLE_STORAGE,
```

in here

https://github.com/lowenware/dotrix/blob/19e8f318eac6d319cc4a651bb6d9e4bd87501a05/dotrix_core/src/renderer/backend.rs#L197-L207

If we don't have access to storage buffers in the vertex shader we will have troubling implementing some of the techniques like storage buffer based culling and single vertex buffer scenes

Reply:
Yes, it is controllable by that feature